analyzers/rolaguard_base_analyzer/BaseAnalyzerMain.py

```diff
@@ -273,13 +279,54 @@ def process_packet(packet, policy):
         device.max_lsnr < policy.get_parameters("LAF-102")["minimum_lsnr"]
     ):
         emit_alert(
-            "LAF-102", 
+            alert_type="LAF-102", 
             packet=packet,
             device=device,
             device_session=device_session,
             gateway=gateway,
             lsnr=device.max_lsnr
         )
 
+    if packet.is_retransmission:
+        # Initialize dict entry if it not exists, increment it otherwise
+        if devices_retransmissions.get(device.id) is None:
```

Comment:
`if device.id in devices_retransmissions:`

Reply:
A cleaner way to check the condition, will do it.